pyproject.toml

```diff
@@ -39,6 +37,7 @@ include-package-data = false
 
 [tool.setuptools.dynamic]
 version = {attr = "web.__version__"}
+dependencies = {file = ["requirements.txt"]}
```

Comment:
Please edit line 95 below to add `notin` so that `codespell` passes.
```
ignore-words-list = "asend,gae,notin"
```

Reply:
OK, done!

Remember -- you can't merge this without changing release.toml and pyproject.yaml from `web.py-update` to be `web.py`, and fixing the version...  And, before tagging the release version pushing the tag, you'll have to fix up the official PyPI to authorize the official webpy repo as a Publisher!